examples/pulse_control/2_Native_gate_calibrations.ipynb

```diff
@@ -0,0 +1,418 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    print(device._native_gate_calibration is None)
</pre><p>Let's try to avoid accessing private attributes in notebooks</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/294/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>